mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageMapperTest.java

```diff
@@ -60,7 +61,7 @@ void findInMailboxLimitShouldLimitProjectionReadCassandraQueries(CassandraCluste
         cassandra.getConf().recordStatements(statementRecorder);
 
         int limit = 2;
-        messageMapper.findInMailbox(benwaInboxMailbox, MessageRange.all(), FetchType.Full, limit);
+        ImmutableList.copyOf(messageMapper.findInMailbox(benwaInboxMailbox, MessageRange.all(), FetchType.Full, limit));
```

Comment:
why copying it since it is not used?

Reply:
To fully consume the (lazy) iterator.

Reply:
maybe leave a comment to explain this in the test.